### ISSUE TITLE
Where is the image training/testing data set?

### ISSUE THREAD

**Issue by liuliuball45** [lmoroney/io19#1]
Thanks for the presentation. Wondering where you have stored those training data set if I would like to reproduce what you have done during the demo at 2019 I/O.

**Comment by tannerdolby** [comment]
Hi there liuliuball45, The training/test set can be found on the first speaker at Google I/O 19' (Laurence Moroney) website. Here is the tensorflow page link that segues to Laurence's webpage with the rps and test-rps-set for your CNN. https://www.tensorflow.org/datasets/catalog/rock_paper_scissors

This is the direct link located on the tensorflow.org page if you face any issues,
 http://laurencemoroney.com/rock-paper-scissors-dataset
